doc/fuzzing.md

```diff
@@ -171,9 +172,9 @@ $ cmake -B build_fuzz \
    -DCMAKE_C_COMPILER="$(pwd)/AFLplusplus/afl-clang-lto" \
```

Comment:
on line 168 we still have `make -C AFLplusplus/ source-only`, is that deliberate?

Reply:
It is related to the building process for the https://github.com/AFLplusplus/AFLplusplus repo, right?

Reply:
My mistake, indeed